SparseGrids/tsgGridFourier.hpp

```diff
@@ -0,0 +1,91 @@
+
+#ifndef __TASMANIAN_SPARSE_GRID_FOURIER_HPP
+#define __TASMANIAN_SPARSE_GRID_FOURIER_HPP
+
+#include <cstdlib>
+#include <math.h>
+#include <complex.h>
+
+#include "tsgEnumerates.hpp"
+#include "tsgIndexSets.hpp"
+#include "tsgCoreOneDimensional.hpp"
+#include "tsgIndexManipulator.hpp"
+#include "tsgLinearSolvers.hpp"
+#include "tsgOneDimensionalWrapper.hpp"
+#include "tsgGridCore.hpp"
+
+#include "tsgAcceleratedDataStructures.hpp"
+
+namespace TasGrid{
+
+class GridFourier{
```

Comment:
* must inherit from `BaseCanonicalGrid`
* must also add multiple `evaluate()` functions, you can make those fallback to basic `evaluate()` and `evaluateBatch()`